src/ucp/rndv/rndv.c

```diff
@@ -1091,6 +1122,12 @@ ucp_rndv_recv_start_get_pipeline(ucp_worker_h worker, ucp_request_t *rndv_req,
     ucp_rndv_req_init_zcopy_lane_map(rndv_req, rndv_req->send.mem_type,
                                      UCP_REQUEST_SEND_PROTO_RNDV_GET);
 
+    lane = ucp_rndv_zcopy_get_lane(rndv_req, &uct_rkey,
+                                   UCP_REQUEST_SEND_PROTO_RNDV_GET);
+    if (lane == UCP_NULL_LANE) {
```

Comment:
Don't we already check it inside ucp_rndv_progress_rma_get_zcopy, called from progress, as set by ucp_rndv_send_frag_get_mem_type in line 1137?
We can check if lanes are valid inside ucp_rndv_progress_rma_get_zcopy, or before it, but why need both?

Reply:
removed it from `ucp_rndv_progress_rma_get_zcopy()` - just added assertion to check that it is called when there are lanes to do GET Zcopy.

Reply:
upd: unfortunately, i found out that we still need this code to get GET Zcopy lane to get correct UCT rkey.
but it `if (lane != UCP_NULL_LANE)` replaced with assertion